### PR TITLE
Add go cross compilers and powerpc64{,le} gccgo

### PR DESCRIPTION
**Pull request by grooverdan**

### CODE REVIEW

File: etc/config/go.amazon.properties [hunk 1]
```diff
@@ -52,6 +52,24 @@ compiler.gl1110.exe=/opt/compiler-explorer/golang-1.11/go/bin/go
...
+group.cross.supportsBinary=false
+group.cross.groupName=Cross Fortran
+
+###############################
+# GCC for PPC
+group.ppc.compilers=ppc64leg8:ppc64g8
+group.ppc.groupName=POWER Compilers
+group.ppc.isSemVer=true
+compiler.ppc64leg8.exe=/opt/compiler-explorer/powerpc64le/gcc-at12/powerpc64le-unknown-linux-gnu/bin/powerpc64le-unknown-linux-gnu-gccgo
+compiler.ppc64leg8.name=power64le Advance Toolchain 12.0 (gcc branch ibm/gcc-8-branch)
```
Comment (RabsRincon) [inline etc/config/go.amazon.properties]: I think the names for this compilers might be a bit too long, but nothing too worriesome. If you could come up with a shorter version (Are all the naming details needed to properly differenciate them? Or could some of that info be just, if needed, gathered from 
imagen which shows the version as reported by the compiler version flag?)

Comment (grooverdan) [reply etc/config/go.amazon.properties]: Thanks for review. Name shorted.. Quite right, the version flag is:
<pre>
$ powerpc64le-unknown-linux-gnu-gccgo --version
powerpc64le-unknown-linux-gnu-gccgo (crosstool-NG 1.23.0.471-ad86) 8.2.1 20180727 Advance Toolchain AT12, based on subversion id 263037
</pre>

**Comment by mattgodbolt** [comment]
Hey @grooverdan - I'm sorry I didn't get time to look at this today: I cleared some other PRs. I can't remember quite how we left this, but I seem to recall at least one of the builds not working...I will do my best to look at this later in the week!

**Comment by grooverdan** [comment]
compiler-explorer-images  - crosstool-ng - older versions (1.22/1.23) need to have Docker image based on ubuntu:16.04 to build. Newer toolchain cross builds will build with compiler-explorer-image:master Ubuntu-18.04 based builder.

**Comment by mattgodbolt** [comment]
Got it. Thanks for the quick reply! I'll see what I can rustle up.

**Comment by mattgodbolt** [comment]
@grooverdan I've been able to build all the cross-compilers (yay) and so am working on merging them in. Thanks again for your help and patience!

**Pull request merged**